### PR TITLE
Add a metadata field which isn't validated.

### PR DESCRIPTION
**Pull request by almet**

### CODE REVIEW

File: daybed/schemas/validators.py [hunk 1]
```diff
@@ -32,7 +32,8 @@ def __init__(self, definition):
         for field in definition['fields']:
             field['root'] = self
             fieldtype = field.pop('type')
-            self.add(registry.validation(fieldtype, **field))
+            if fieldtype is not 'metadata':
+                self.add(registry.validation(fieldtype, **field))
```
Comment (leplatrem) [inline daybed/schemas/validators.py]: Why was this necessary ? If a metadata has no specific validation, a colander Mapping should be quite flexible enough IMO

**Comment by leplatrem** [comment]
Eventually, with @Natim example, I could get what you want to achieve !

Maybe the name `metadata` is not totally obvious... Hard to find better though I must admit :) What about `annotation`, `note`, `comment`... ?

**Comment by almet** [comment]
Yeah obviously this was in need of tests. "annotation" seems to be the best fit here.

**Comment by Natim** [comment]
Ok to change for annotation

**Comment by Natim** [comment]
PR updated.
